### DISCUSSION TITLE
telepresence with cellphone & zoom

### DISCUSSION THREAD

**Discussion by andrit** [iRobotEducation/create3_docs#230] [Questions] [answered]
Hi, I have the Create3 robot and I am planning to create a telepresence project.
I noticed that https://edu.irobot.com/learning-library/draft-of-create-3-teleop-project is now 404.

I would like to create a 3D printed holder for the cellphone, connect it to the robot and use the camera on teh phone to connect to a zoom call. Then use the laptop connected to zoom call to also use the camera to navigate to a subject.

I could use some help with getting this going, including the stl files for the cellphone holder and the rclpy commands to establish this capability.

Is there anything that you can point me to that might help me out?
Thanks

**Comment by shamlian** [comment, marked as answer]
Hi there,

Are you looking for https://edu.irobot.com/learning-library/teleop-twist-with-the-create-3 ?

As far as a cell phone holder, I'd say you should look for one from a free 3D model repository like Thingiverse, Cults, or Printables that works for your device, and then work to add holes (digitally or manually) to mount to the regular pattern on the robot's faceplate. See [this page](https://iroboteducation.github.io/create3_docs/hw/mechanical/#payload) for ideas on how to mount stuff to the robot.

Not sure how helpful this is, but hope it gets you some of the way there.

**Reply by andrit** [reply]
Yes, thats the link Im looking for.
That should get me to the next steps.
Thank you!
